fmriprep/cli/run.py

```diff
@@ -81,6 +82,8 @@ def get_parser():
     # Re-enable when option is actually implemented
     # g_bids.add_argument('-r', '--run-id', action='store', default='single_run',
     #                     help='select a specific run to be processed')
+    g_bids.add_argument('--bids_filters', action='store', type=Path,
```

Comment:
We've generally been moving to hyphen-separated words in arguments. It will still show up as `opts.bids_filters` later.

```suggestion
    g_bids.add_argument('--bids-filters', action='store', type=Path,
```